challenges/desafio7.js

```diff
@@ -0,0 +1 @@
+db.voos.find({"ano": {$lt:2017}}).count();
```

Comment:
Aqui também dá pra usar apenas o count: 
```suggestion
db.voos.count({"ano": {$lt:2017}});
```